### ISSUE TITLE
[BUG/Chrome] Nitter & Tweetdeck

### ISSUE THREAD

**Issue by barbiehunter** [libredirect/libredirect#186]
Good day,
I use Brave and use the chromium version, but I notice an error when using Tweetdeck. When you use Tweetdeck and want to open a link, it is not possible to Nitter (see screenshot). To see if it is a general problem I tried the browser "Librewolf" under Tweetdeck and there it works without problems.

![image](https://user-images.githubusercontent.com/38357837/162943194-8d1863e6-18d7-487b-9254-e0b1b571afe3.png)

**Comment by ManeraKai** [comment]
examples?

**Comment by barbiehunter** [comment]
> Examples?

What for Examples you mean? if i go go as example https://twitter.com/Twitch/status/1513620141669376001 i get the Error as above (the screenshot in my first message)
Video as Example:
https://user-images.githubusercontent.com/38357837/162946447-c843e907-b875-407f-ae3a-b2226db72830.mp4

**Comment by ManeraKai** [comment]
Sorry for confusion, I meant an example for a TweetDeck link.

**Comment by barbiehunter** [comment]
> Sorry for confusion, I meant an example for a TweetDeck link.

Tweetdeck is https://tweetdeck.twitter.com/ for a better Organistation about your Timeline etc. it worked like 4-5 Days ago but since then it dont want to work anymore with Libredirect

**Comment by ManeraKai** [comment]
Which version are you running, it it an old one?

**Comment by EdwardLangdon** [comment]
@ManeraKai I think the Regex is causeing the problem, Might be allowing an Exception for Tweetdeck would be better. Because Tweetdeck is totally different from Normal Twitter.

**Comment by ManeraKai** [comment]
I'm just confused. I didn't use TweetDeck from a long time so I don't remember much. @barbiehunter is it that if you paste this link (https://twitter.com/Twitch/status/1513620141669376001) it won't work? or when you click a link in TweetDeck?

**Comment by ManeraKai** [comment]
@EdwardLangdon  use tweetdeck

**Comment by barbiehunter** [comment]
> I'm just confused. I didn't use TweetDeck from a long time so I don't remember much. @barbiehunter is it that if you paste this link (https://twitter.com/Twitch/status/1513620141669376001) it won't work? or when you click a link in TweetDeck?

It happens when i want to open a Link in Tweetdeck. I Disabled all my other Extentions too to make sure its not a issue of me myself. ( as Example https://user-images.githubusercontent.com/38357837/162946447-c843e907-b875-407f-ae3a-b2226db72830.mp4 )

**Comment by EdwardLangdon** [comment]
> @EdwardLangdon use TweetDeck

I cannot. There are some complications in it. Sorry.

**Comment by barbiehunter** [comment]
It looks like it dont want to work with Tweetdeck itself (Video as Example again). Im confused myself, because somtimes i have the issue of Random Normal Tweets.

https://user-images.githubusercontent.com/38357837/162953462-c8b34f9c-7b92-421f-9cfa-0980a5a3c7a4.mp4

**Comment by ManeraKai** [comment]
I don't think it's related to TweetDeck. Maybe related to this issue: #110

**Comment by barbiehunter** [comment]
> I don't think it's related to TweetDeck. Maybe related to this issue: #110

I did what he did with "clear site data" but dont Fixed the issue. The only thing that i Noticed if i clear the site data and then open Tweetdeck i get Redirect to a nitter website. but when i disabled it and log back in and enable it again i get the same issue.

**Comment by ManeraKai** [comment]
I will release a new version that has an exception feature fixed. I'll tell you when I release to test adding TweetDeck to the exception list.

**Comment by barbiehunter** [comment]
But may a issue to fix too would be a Playback issues and Profile Pictures (Check the Screenshot). 
![image](https://user-images.githubusercontent.com/38357837/162964890-6cc82dfa-5e9a-48d6-904f-0fb6379d3ff2.png)

**Comment by barbiehunter** [comment]
> This is because tweetdeck is using twitter for requests and they're being redirected so tweetdeck won't accept them.

Ouh ok then. i will let this comment open to check if i get the Tweetdeck issue Later again and if i do i will send the Error code.

**Comment by barbiehunter** [comment]
A Friend tried it himself and got a error (Screenshot), not sure if thats helpful or something. He dont have the issue on Firefox.
![image](https://user-images.githubusercontent.com/38357837/162971646-7f331c7d-a8ac-4070-9f7e-95ef7221ef3c.png)

**Comment by ManeraKai** [comment]
This was from the twitter tab not from the TweetDeck tab right?

**Comment by barbiehunter** [comment]
> This was from the twitter tab not from the TweetDeck tab right?

Thats right.

**Comment by ManeraKai** [comment]
It's canceling the first request that is the main url. Then you're being left in a twitter page that tries to make requests but everything thing gets redirected. That's why when you refresh the page it redirects normally. As I said in #110, I can't do anything about this. idk why chromium browsers do this.

**Comment by ManeraKai** [comment]
@barbiehunter Released a new version. Add TweetDeck as an exception. You can find this setting in the Generals page.

**Comment by barbiehunter** [comment]
> @barbiehunter Released a new version. Add TweetDeck as an exception. You can find this setting in the Generals page.

Yeah i installed the new version already, Seems like everything is fixed now for this moment. When i get the issue again with Twitter/Tweetdeck i will open a new comment.

**Comment by EdwardLangdon** [comment]
Okay, As it seems fixed. I am closing this. If it isn't we will reopen do not open another. Best regards.

**Comment by barbiehunter** [comment]
Hey im Back with the same issue looks like. I tried this time to get some Screenshots of the error and tried again 3 times the fix of https://github.com/libredirect/libredirect/issues/110 and still same issue.  Tried to take some screenshots after Removing and Load the Plugin again aswell.
![image](https://user-images.githubusercontent.com/38357837/165639734-93578073-f7f3-4579-9d13-6f8640e34ec3.png)
![image](https://user-images.githubusercontent.com/38357837/165639744-3838289a-c86e-4dfd-99c7-999b6c039e73.png)
![image](https://user-images.githubusercontent.com/38357837/165640578-bca0667a-6f6e-48a6-9c73-1b0f63cf125e.png)
